src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Consider removing the 'no-explicit-any' eslint rule disabling and use a more specific type instead of 'any' in your data parameter for the request function.